main.go

```diff
@@ -337,17 +350,35 @@ func sendEmail(event *corev2.Event) error {
 	return conn.Quit()
 }
 
-func resolveTemplate(templateValue string, event *corev2.Event) (string, error) {
-	var resolved bytes.Buffer
-	tmpl, err := template.New("test").Parse(templateValue)
+func resolveTemplate(templateValue string, event *corev2.Event, contentType string) (string, error) {
+	var (
+		resolved bytes.Buffer
+		tmpl     templater
+		err      error
+	)
+	if contentType == ContentHTML {
+		// parse using html/template
+		tmpl, err = htemplate.New("test").Parse(templateValue)
+	} else {
+		// default parse using text/template
+		tmpl, err = ttemplate.New("test").Parse(templateValue)
+	}
+
 	if err != nil {
 		return "", err
 	}
+
 	err = tmpl.Execute(&resolved, *event)
 	if err != nil {
 		return "", err
 	}
 
+	// \n replacement must be done after tmpl.Execute to catch
+	// template values
+	if contentType == ContentHTML {
+		return strings.Replace(resolved.String(), "\n", "<br>", -1), nil
+	}
+
```

Comment:
I wonder if this is really desirable? It seems like this could result in too many line breaks in a lot of situations.

For instance, I could do this, expecting it to only result in a single line of rendered HTML text content for each item:
```
{{ range $i, $e := .Items }}
<div>
    <span>It looks like item number {{ $i }} is {{ $e }}.</span>
</div>
{{ end }}
```

In this template, replacing newlines with `<br>` would inject a lot of line breaks that were never intended.

Reply:
I see, thanks for the clarification. In that case I'm happy to merge the PR when conflicts are resolved. Thanks!